docs/internal/bench2.rst

```diff
@@ -741,6 +750,11 @@ This is a subset of the options. Use ``--help`` option to see all the options.
     Overwrite individual worker configs to start their test actions (writes &
     forwards) N seconds from now (absolute time reference)
 
+.. option:: --tag TAG
```

Comment:
```suggestion
.. option:: --tag TAG

```

I don't know why but it gets messed up if there isn't a blank line:
![Screenshot from 2020-11-13 01-07-21](https://user-images.githubusercontent.com/5941194/99039335-ca7f1380-254c-11eb-8e19-e1b79d69097f.png)